### DISCUSSION TITLE
shortcuts for pixel editor

### DISCUSSION THREAD

**Discussion by skjoldbroder** [nesbox/TIC-80#1418] [Ideas]
Hi, 

When creating art in the built-in pixel editor, it would be helpful to have shortcuts to move 1 step forwards, and 1 step backwards in the palette. I would suggest two keys located near the arrow keys, since your fingers are usually resting on those to be able to navigate around the sprites when drawing. 

This feature would make a big difference e.g. when antialiasing, because you are constantly stepping back and forwards between 2-3-4 shades. It would also just make it faster to draw in general, since selecting colours could be done without taking your hands off the keyboard.

all the best
/M

**Comment by RobLoach** [comment]
I think this is a great idea... In [Aseprite](https://www.aseprite.org/), it's left and right arrows.

**Reply by joshgoebel** [reply]
Great idea. Though I think best tools will always be an external editor and import/export.

**Reply by skjoldbroder** [reply]
@joshgoebel true.. I'm thinking of going that route as well :)

**Reply by nesbox** [reply]
Hi,
Just checked, Aseprite uses `[` and `]` keys to navigate in the palette.
We can add this without problems.
Thanks.

**Reply by nesbox** [reply]
Created the issue here #1421

**Reply by nesbox** [reply]
We have palette navigation shortcuts on the latest build.
https://github.com/nesbox/TIC-80/actions/runs/762915109
Thanks.
